### ISSUE TITLE
ChemicalEntity (PUBCHEM:644073) <related_to> BiologicalProcessOrActivity, Gene, Pathway <related_to> DiseaseOrPhenotypicFeature (MONDO:000235)

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/testing#187]
**Query:** [bupeDental.json](https://github.com/NCATSTranslator/testing/blob/main/ars-requests/not-none/1.2/bupeDental.json)
**PK:** [8c6ea099-d82e-46b4-807f-c57fde291736](https://arax.ncats.io/?r=8c6ea099-d82e-46b4-807f-c57fde291736)

Looking for ideas of how to represent oral PH

![image](https://user-images.githubusercontent.com/38321826/161638851-dc320e6f-b166-45d6-9a72-89ca5757fe0e.png)

**Comment by colleenXu** [comment]
Notes:
* Does "oral pH" refer to acidity in the mouth (like saliva acidity)? I found these two terms related to saliva pH measurements, with x-refs to LOINC codes. However, I'm not sure how many knowledge sources would have these terms...[pH:LsCnc:Pt:Saliva:Qn](https://uts.nlm.nih.gov/uts/umls/concept/C2735242) and [pH | Saliva | Chemistry - non-challenge](https://uts.nlm.nih.gov/uts/umls/concept/C5176838)
* I tried browsing this topic...It seemed like there was a lack of information + a bunch of reasoning that I'm not sure this tool could handle:
    * I could only find very limited info on how acidic Buprenorphine is
    * I'm using a bunch of reasoning that I'm not sure this tool can use. 
        * context of taking sublingual buprenorphine: it seems to involve holding it in the mouth for several min - 15 min. And patients were supposedly told not to drink for 15-30 min after taking it (vs rinsing the mouth right after would help if it was very acidic)
        * what side effects are occurring more in patients taking sublingual Buprenorphine vs other routes of administration (ones used for long-term treatment...oral?)? The xerostomia (less saliva / dry mouth) likely occurs in both cases, but the "exposing the inside of the mouth to probably-acidic stuff for a long time" happens only with the sublingual... 
        * related to the above point, how do we tell apart stuff that happens "due to xerostomia" vs what happens with "long term exposure to acidic stuff"?
        * Are the side effects of sublingual buprenorphine similar to that of [other conditions that expose the inside of the mouth to "acidic stuff" for a long time](https://en.wikipedia.org/wiki/Acid_erosion)? Those conditions include GERD, stuff with vomiting (bulimia, chronic alcoholism, even pregnancy), and stuff like "drinking lots of soda". 

There's a lot of terms related to symptoms that could be used in queries:
* [Tooth Erosion](https://uts.nlm.nih.gov/uts/umls/concept/C0040436)
* [acid erosion of teeth](https://uts.nlm.nih.gov/uts/umls/concept/C2146654)
* Burning mouth IDs ([HPO](https://hpo.jax.org/app/browse/term/HP:0032143), [UMLS](https://uts.nlm.nih.gov/uts/umls/concept/C0877717))
* [Fragile teeth](https://hpo.jax.org/app/browse/term/HP:0025124), [Premature loss of permanent teeth](https://hpo.jax.org/app/browse/term/HP:0006357), [Tooth abscess](https://hpo.jax.org/app/browse/term/HP:0030757)
* [dental cavities / caries](https://hpo.jax.org/app/browse/term/HP:0000670)
* related symptoms: [bad breath](https://hpo.jax.org/app/browse/term/HP:0100812), [Oral cavity bleeding](https://hpo.jax.org/app/browse/term/HP:0030140), [Oral bleeding](https://hpo.jax.org/app/browse/term/HP:0040184)
* saliva stuff: [Abnormality of salivation](https://hpo.jax.org/app/browse/term/HP:0100755), [Xerostomia](https://hpo.jax.org/app/browse/term/HP:0000217)
